tests/cases/find_entry_point_plugins_test.py

```diff
@@ -112,7 +113,9 @@ def resource_stream_json_value():
         failures = getPluginFailureInfo()
         self.assertIn('entry_point_plugin_bad_json', failures)
         self.assertIn('traceback', failures['entry_point_plugin_bad_json'])
-        self.assertIn('ValueError', failures['entry_point_plugin_bad_json']['traceback'])
+        self.assertIn(
+            'JSONDecodeError' if sys.version_info >= (3, 5) else 'ValueError',
```

Comment:
This is probably the only spot where you need to differentiate `JSONDecodeError` from `ValueError`.

Reply:
Maybe, see above.